cmd/trust-manager/app/app.go

```diff
@@ -95,48 +94,34 @@ func NewCommand() *cobra.Command {
 				Cache: cache.Options{
 					ReaderFailOnMissingInformer: true,
 					ByObject: map[client.Object]cache.ByObject{
+						// Cache metadata for ConfigMaps cluster-wide
+						&metav1.PartialObjectMetadata{
+							TypeMeta: metav1.TypeMeta{
```

Comment:
I fear that controller runtime does not differentiate between the partial metadata object and the typed object. There will be a config collision.

Reply:
See #172

Reply:
This is the c/r code: https://github.com/kubernetes-sigs/controller-runtime/blob/v0.16.2/pkg/cache/cache.go#L304-L316